### PR TITLE
Accessibility Checks

### PR DESCRIPTION
**Pull request by asharanjith**
Page title checks: No issue found

Image text alternatives ("alt text"): Add alt text to the project cards image. 
https://github.com/asharanjith/Portfolio-Asha/blob/33e5513a84772c3b7dc894ba7c2b8f092424f491/index.html#L56-L57

Headings: No issue found.

Contrast checks: Accurate.

Resize text: Enlarge (Ctrl  +) and minimize (Ctrl -) working properly.

Keyboard access and visual focus: Tab around is moving in order.

Moving, Flashing, or Blinking Content: No issue found.

Multimedia: Not applicable

Basic structure check: No issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,8 +53,8 @@ <h4 class="media-text">LET’S CONNECT</h4>
             <div class="sample-project project-normal">
                 <div class="project-header ">
                     <div class="image-holder">
-                        <img src="./Assets/SnapshootPortfolio.png" alt="header" class="header-img mobile"/>
-                        <img src="./Desktop/SnapshootDesktop.svg" alt="header" class="header-img desktop"/>
+                        <img src="./Assets/SnapshootPortfolio.png" alt="header" class="header-img mobile" alt="Project one Sample Image"/>
+                        <img src="./Desktop/SnapshootDesktop.svg" alt="header" class="header-img desktop" alt="Project one Sample Image"/>
```
Comment (cynthiainga) [inline index.html]: - [ ] Good job by adding the `alt` but for best practice of HTML the image should have only one attribute and not two, please kindly fix it 🙃 🙃

File: index.html [hunk 2]
```diff
@@ -92,8 +92,8 @@ <h4 class="description-title">Tonic</h4>
             <div class="sample-project project-reverse row">
                 <div class="project-header ">
                     <div class="image-holder bg">
-                        <img src="./Assets/SnapshootPortfolio1.png" alt="header" class="header-img mobile"/>
-                        <img src="./Desktop/SnapshootDesktop2.svg" alt="header" class="header-img desktop"/>
+                        <img src="./Assets/SnapshootPortfolio1.png" alt="header" class="header-img mobile" alt="Project two Sample Image"/>
+                        <img src="./Desktop/SnapshootDesktop2.svg" alt="header" class="header-img desktop" alt="Project two Sample Image"/>
```
Comment (cynthiainga) [inline index.html]: - [ ] Same remark about attributes, please kindly make sure your images have one `alt` 😉 😉

File: index.html [hunk 3]
```diff
@@ -139,8 +139,8 @@ <h4 class="description-title">Multi-Post Stories</h4>
             <div class="sample-project project-normal">
                 <div class="project-header ">
                     <div class="image-holder bg">
-                        <img src="./Assets/SnapshootPortfolio2.png" alt="header" class="header-img mobile"/>
-                        <img src="./Desktop/SnapshootDesktop3.svg" alt="header" class="header-img desktop"/>
+                        <img src="./Assets/SnapshootPortfolio2.png" alt="header" class="header-img mobile" alt="Project three Sample Image"/>
+                        <img src="./Desktop/SnapshootDesktop3.svg" alt="header" class="header-img desktop" alt="Project three Sample Image"/>
```
Comment (cynthiainga) [inline index.html]: - [ ] Same remark, please kindly fix it 👍

File: index.html [hunk 4]
```diff
@@ -187,8 +187,8 @@ <h4 class="description-title desktop">Facebook 360</h4>
             <div class="sample-project project-reverse">
                 <div class="project-header">
                     <div class="image-holder bg">
-                        <img src="./Assets/SnapshootPortfolio3.png" alt="header" class="header-img mobile"/>
-                        <img src="./Desktop/SnapshootDesktop4.svg" alt="header" class="header-img desktop"/>
+                        <img src="./Assets/SnapshootPortfolio3.png" alt="header" class="header-img mobile" alt="Project four Sample Image"/>
+                        <img src="./Desktop/SnapshootDesktop4.svg" alt="header" class="header-img desktop" alt="Project four Sample Image"/>
```
Comment (cynthiainga) [inline index.html]: - [ ] The same here, make sure your images have one `alt` for best practice 😁 😁

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @asharanjith, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing 👍 
- [ ] Descriptive PR ✔️ 
- [ ] Used correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- You can add the `Getting started ` part to your `README` file and explain how users can clone and open your project locally and also what they need to open your project in their device 😉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @asharanjith , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing the required changes! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**